src/Microsoft.ML.LightGbm/LightGbmCatalog.cs

```diff
@@ -119,6 +136,22 @@ public static class LightGbmExtensions
             return new LightGbmBinaryTrainer(env, options);
         }
 
+        /// <summary>
+        /// Create <see cref="LightGbmBinaryTrainer"/> from a pre-trained LightGBM model, which predicts a target using a gradient boosting decision tree binary classification.
+        /// </summary>
+        /// <param name="catalog">The <see cref="BinaryClassificationCatalog"/>.</param>
+        /// <param name="lightGbmModel"> A pre-trained <see cref="System.IO.Stream"/> of a LightGBM model file inferencing</param>
+        /// <param name="featureColumnName">The name of the feature column. The column data must be a known-sized vector of <see cref="System.Single"/>.</param>
+        public static LightGbmBinaryTrainer LightGbm(this BinaryClassificationCatalog.BinaryClassificationTrainers catalog,
```

Comment:
Will the trainer be able to re-fit after loading from an existing model file.

Reply:
Currently the answer is no. Thats something we could look at doing though in the future.